galley/pkg/crd/validation/webhook.go

```diff
@@ -504,3 +529,22 @@ func (wh *Webhook) admitMixer(request *admissionv1beta1.AdmissionRequest) *admis
 	reportValidationPass(request)
 	return &admissionv1beta1.AdmissionResponse{Allowed: true}
 }
+
+func checkFields(raw []byte, kind string, namespace string, name string) (string, error) {
+	trial := make(map[string]json.RawMessage)
+	if err := yaml.Unmarshal(raw, &trial); err != nil {
+		scope.Infof("cannot re-decode configuration: %v", err)
```

Comment:
"re-decode" doesn't seem like the right error message here. Maybe something about not being able to decode the top-level well-known fields?